File: app/src/test/java/com/mycompany/myapp/data/api/github/model/CommitTestHelper.java
package com.mycompany.myapp.data.api.github.model;

public class CommitTestHelper {
    public static Commit stubCommit(String authorName, String message) {
        Author author = new Author(authorName);
        CommitDetails commitDetails = new CommitDetails(message, author);
        return new Commit(commitDetails);
    }
}

File: app/src/test/java/com/mycompany/myapp/data/api/github/model/CommitTestHelper.kt
package com.mycompany.myapp.data.api.github.model

object CommitTestHelper {
    fun stubCommit(authorName: String, message: String): Commit {
        val author = Author(authorName)
        val commitDetails = CommitDetails(message, author)
        return Commit(commitDetails)
    }
}
